geode-core/src/main/java/org/apache/geode/distributed/AbstractLauncher.java

```diff
@@ -795,25 +785,25 @@ protected String toString(final Date dateTime) {
 
     // the value of a Number as a String, or "" if null
     protected String toString(final Number value) {
-      return StringUtils.defaultString(value);
+      return defaultString(value);
     }
 
     // a String concatenation of all values separated by " "
     protected String toString(final Object... values) {
-      return values == null ? "" : StringUtils.join(values, " ");
+      return values == null ? "" : join(values, " ");
     }
 
     // the value of the String, or "" if value is null
     protected String toString(final String value) {
-      return ObjectUtils.defaultIfNull(value, "");
+      return defaultIfNull(value, "");
     }
   }
 
   /**
    * The Status enumerated type represents the various lifecycle states of a GemFire service (such
    * as a Cache Server, a Locator or a Manager).
    */
-  public static enum Status {
```

Comment:
It's static with or without the keyword. IntelliJ grays it out so I just deleted it. Similar to public on methods defined in an Interface.